packages/core/src/fields/types/relationship/views/RelationshipSelect.tsx

```diff
@@ -99,6 +98,40 @@ export function useFilter (value: string, list: ListMeta, searchFields: string[]
 
     for (const fieldKey of searchFields) {
       const field = list.fields[fieldKey]
+
+      // @ts-expect-error TODO: fix fieldMeta type for relationship fields
+      if (field.fieldMeta?.refSearchFields) {
```

Comment:
@pnodet I merged this into the pre-exitsing `searchFields` query building code, less type passing